### PR TITLE
Add Hawkular to the list of supported tracers

### PR DESCRIPTION
**Pull request by bensigelman**
cc: @objectiser -- thanks for doing the work on your end!

### CODE REVIEW

File: _docs/pages/supported-tracers.md [hunk 1]
```diff
@@ -33,3 +33,7 @@ For more details, read [the godocs](https://godoc.org/github.com/sourcegraph/app
 ## LightStep
```
Comment (yurishkuro) [inline _docs/pages/supported-tracers.md]: We should link to this page from the section header on the landing page

**Comment by yurishkuro** [comment]
LGTM, but has a conflict

**Comment by pritianka** [comment]
Hi @objectiser - Thanks for your work on this! Would you be open to chatting about Hawkular for the OpenTracing blog? An interview-style post would make most sense but am happy to do a different format if you prefer.

File: _docs/pages/supported-tracers.md (hunk 1 above)
Comment (bensigelman) [inline _docs/pages/supported-tracers.md]: @yurishkuro do you want Jaeger to be its own subsection, independent of Zipkin?

**Comment by bensigelman** [comment]
This ended up being non-trivial css-wise for two reasons:

1. The number of tracer impls is large enough that they don't all fit at many screen resolutions
1. @yurishkuro it's actually pretty tricky to link from the "Supported Tracers" thing because it's part of a `:before` thing, not normal doc contents (i.e., it's in the css, not the html).

@reimertz would you be willing to adopt this branch and fix the problems above? Thanks!

**Comment by pritianka** [comment]
Since the list of tracers (and hopefully the projects instrumented for OT
as well soon) is increasing, I'd like to revisit the idea of a separate
page that we can link to from the homepage after we've displayed the max
number of logos possible there. This page can have some detail around each
tracer and links to documentation/repos.


ᐧ

Priyanka Sharma
Advisor, HeavyBit <http://heavybit.com/>
Head of Marketing & Partnerships, LightStep <http://lightstep.com/>
Co-founder, WakaTime <http://www.wakatime.com/> - Quantify your coding
www.twitter.com/pritianka
650-796-7125

**Comment by bensigelman** [comment]
@pritianka that page already exists, actually :)

http://opentracing.io/documentation/pages/supported-tracers

Maybe we should choose the ~4-5 that are most actively developed for the homepage... seems like a fine solution to me. Thoughts from others? (@yurishkuro?)

**Comment by pritianka** [comment]
Oh right - sorry, I confused this with framework and companies pages that we didn't do last time. 

The word tracers on the homepage can link to this page and we can have all the tracers listed here and only the most active ones can be on the homepage?

**Comment by yurishkuro** [comment]
"Most active" is hard to measure objectively. I'd rather list all until this proves to be a problem.

**Comment by objectiser** [comment]
@bensigelman Would you like me to provide some text for the supported tracers page aswell?

@pritianka Yes would be interested - could you contact me at gbrown@redhat.com so we can discuss details.

**Comment by pritianka** [comment]
Sent you a note @objectiser. Talk to you soon!

**Comment by wu-sheng** [comment]
I think we should create a new section of Index Page, like `Why OpenTracing` section, and just below it. And we can layout many supported Tracers and supported Frameworks. Like a **OpenTracing Supporter Wall**.

**Comment by reimertz** [comment]
@benaigelman what do you think about inviting @fractaly to this discussion, maybe she could help us figuring out how to fit n supporters / frameworks on the site with a new design if we prefer to show all of them on the landing page? 

I see both pros and cons of this, a supporters list could get very long, to the point where it will be messy to show all of them, but on the other hand could be solved with some semi-fancy carousel etc. 

Once we have a firm decision on how this should be solved,  I'll start working on it asap. :)

**Comment by bensigelman** [comment]
@reimertz sure, I'd be glad to have @fractaly's help! And a carousel is a fine way to deal with the problem. For a while there won't be an overwhelming number of these implementations... probably no more than 10 for the next year or so, I'd guess/hope?

**Comment by fractaly** [comment]
Yeah, I agree a carousel makes sense. Especially when you get those first new tracers/frameworks that creates a new line, but it doesn't really look good to add another line yet.. I also think it would make sense to make the background divider be centered, to get an even amount of space for both tracers and frameworks. That will probably open up for about 6 logos in each section on a standard desktop view, and maybe 4-5 on mobile.

**Comment by bensigelman** [comment]
Alright. I'm going to do something "pragmatic" to list Hawkular. Once there's a functional carousel, it will look better :)

**Comment by wu-sheng** [comment]
Looking forward the `carousel`. Hope I can catch up to add sky-walking on it.     :) :P

Too busy these two weeks.

**Comment by bensigelman** [comment]
@wu-sheng sounds good!

**Comment by bensigelman** [comment]
I removed Hawkular from the OT landing page until we have a carousel to use there. I kept the changes to the "supported tracers" page, of course. Merging this PR for the time being...

**Comment by wu-sheng** [comment]
@bensigelman `HAWKULAR` already in landing page. But it cause the page very *ugly*

It is like a char seq, not several tracer icons or names.

qq20161117-0

**Comment by bensigelman** [comment]
@wu-sheng argh, thank you, yes... I neglected to push a local change before merging. Fixed in master.

**Comment by reimertz** [comment]
Will start implementing the carousel, but push a quick fix to make stuff look good with one more tracer. :)

**Pull request merged**